core/org.osate.xtext.aadl2.ui/src/org/osate/xtext/aadl2/ui/outline/Aadl2OutlineTreeProvider.java

```diff
@@ -140,7 +146,9 @@ protected boolean isLeaf(final EObject modelElement) {
 				|| modelElement instanceof FlowSpecification || modelElement instanceof FlowImplementation
 				|| modelElement instanceof EndToEndFlowImpl || modelElement instanceof Property
 				|| modelElement instanceof PropertyConstant || modelElement instanceof PropertyType
-				|| modelElement instanceof Connection) {
+				|| modelElement instanceof DataType || modelElement instanceof AbstractSubcomponent
+				|| modelElement instanceof Connection || modelElement instanceof ErrorType) {
```

Comment:
Remove ErrorType here. That should be handled in the EMV2 annex

Reply:
I had a hard time trying to find where to add this to in emv2 ... it seems like everything under emv2 is related to analysis. Do you have an idea where I should look? I can look again too and see if I can find it

Reply:
The org.osate.xtext.aadl.errormodel.* plugins contain the code generated from the Xtext grammar. Look in the ...ui plugin.